### ISSUE TITLE
ERROR in r-devel-fedora-clang

### ISSUE THREAD

**Issue by Enchufa2** [r-simmer/simmer.plot#4]
New problems. Limited to the "to" comparison in certain graphs:

```
     1. Failure: a rollback with variable amount is correctly converted to graph (@test-plot-trajectory.R#35) 
     all(graph$edges_df$to == to) isn't true.
     
     
     2. Failure: a rollback with variable amount is correctly converted to graph (@test-plot-trajectory.R#36) 
     all(graph$edges_df$to == to) isn't true.
     
     
     3. Failure: a complex trajectory is correctly converted to graph (@test-plot-trajectory.R#65) 
     all(graph$edges_df$to == to) isn't true.
     
     
     testthat results ================================================================
     OK: 86 SKIPPED: 0 FAILED: 3
     1. Failure: a rollback with variable amount is correctly converted to graph (@test-plot-trajectory.R#35) 
     2. Failure: a rollback with variable amount is correctly converted to graph (@test-plot-trajectory.R#36) 
     3. Failure: a complex trajectory is correctly converted to graph (@test-plot-trajectory.R#65) 
```

**Comment by Enchufa2** [comment]
Solved for Mac, but not for Fedora. I'm not able to reproduce this though:

```
  > test_check("simmer.plot")
  1. Failure: a rollback with variable amount is correctly converted to graph (@test-plot-trajectory.R#38) 
  edges_df$to not equal to c(2, 1).
  1/2 mismatches
  [2] 2 - 1 == 1
  
  
  2. Failure: a rollback with variable amount is correctly converted to graph (@test-plot-trajectory.R#39) 
  edges_df$to not equal to c(2, 1).
  1/2 mismatches
  [2] 2 - 1 == 1
  
  
  3. Failure: a complex trajectory is correctly converted to graph (@test-plot-trajectory.R#68) 
  edges_df$to not equal to c(...).
  1/22 mismatches
  [19] 15 - 1 == 14
  
  
  testthat results ================================================================
  OK: 86 SKIPPED: 0 FAILED: 3
  1. Failure: a rollback with variable amount is correctly converted to graph (@test-plot-trajectory.R#38) 
  2. Failure: a rollback with variable amount is correctly converted to graph (@test-plot-trajectory.R#39) 
  3. Failure: a complex trajectory is correctly converted to graph (@test-plot-trajectory.R#68) 
```

**Comment by Enchufa2** [comment]
The ERROR in SPARC was due to RcppCore/Rcpp#647. Solved in RcppCore/Rcpp#649.
